### PR TITLE
stripping out new analytics to return things back to normal closes #4315

### LINKED ISSUE #4315: 2.2.3 appearing strangely in analytics
At first glance it looks like it's almost not appearing at all, but it looks like GA might just handle "non-interaction" hits very differently. Here's an example:

For the 2.2.3 hash, GA is showing 29 sessions, but those apparently account for 66% of total sessions and 14,065 new users! I think we have to revert some of the 2.2.x changes to get back to normal land with the data.

### PR DESCRIPTION
**Pull request by myleshorton**

### CODE REVIEW

**Comment by myleshorton** [comment]
This just strips out most of the changes in 2.2.x to return analytics back to their previous state. This reflects several things, including that non-interaction hits are handled very differently (apparently) and that the Lantern version actually proved to be very useful in identifying fake versions.

**Comment by myleshorton** [comment]
Hey @oxtoacart you mind taking a look at this -- I just realized our analytics are still pretty messed up!

**Comment by myleshorton** [comment]
It looks like the coverage changes are just based on randomly touched code in the tests @oxtoacart so tempted to ignore.

File: src/github.com/getlantern/flashlight/analytics/analytics.go [hunk 1]
```diff
@@ -33,51 +32,40 @@ var (
...
+	return start(deviceID, version, geolookup.GetIP, trackSession)
 }
 
 // start starts the GA session with the given data.
-func start(deviceID, version string, ipFunc func(time.Duration) string, uaWait time.Duration,
+func start(deviceID, version string, ipFunc func(time.Duration) string,
 	transport func(string, eventual.Getter)) func() {
 	var addr atomic.Value
 	go func() {
-		logging.AddUserAgentListener(func(agent string) {
```
Comment (oxtoacart) [inline src/github.com/getlantern/flashlight/analytics/analytics.go]: @myleshorton With this change, the AddUserAgentListener capability is no longer used.  Can you remove it and the corresponding test please?

**Comment by myleshorton** [comment]
Yup all set @oxtoacart!

**Comment by oxtoacart** [comment]
Code LGTM, just waiting for tests to pass.

**Comment by myleshorton** [comment]
Thanks @oxtoacart! I'll add this on devel as well.

**Pull request merged**